src/corytortoise/DuelsPE/Main.php

```diff
@@ -78,6 +81,19 @@ public function getPrefix() {
       $prefix = str_replace("&", "§", $prefix);
       return $prefix . " ";
     }
+	    
+    public function addNewCommands(){
+		$this->registerc(['DuelCommand'],new duel($this)); 
```

Comment:
Lol. I think using 2 member functions is unnecessary for one command. It can be registered with one line of code on the onEnable function.